cmd/signature-v4_test.go

```diff
@@ -135,21 +135,7 @@ func TestDoesPresignedSignatureMatch(t *testing.T) {
 			region:   "us-west-1",
 			expected: ErrInvalidAccessKeyID,
 		},
-		// (2) Should error when the payload sha256 doesn't match.
```

Comment:
sha256 verification is not done in this function, just signature verification happens here.